changelog.d/2845.feature.rst

```diff
@@ -0,0 +1,5 @@
+Added New commands to Economy
+
+- ``[p]bank prune user`` - This will delete a users account
+- ``[p]bank prune local`` - This will prune a local bank from users who are no longer in the server
```

Comment:
This should specify that this will prune the bank *of accounts* from users no longer in the server.